sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -37,6 +37,33 @@ For any DPC++-related commit, the `[SYCL]` tag should be present in the
 commit message title. To a reasonable extent, additional tags can be used
 to signify the component changed, e.g.: `[UR]`, `[CUDA]`, `[Doc]`.
 
+## Release notes
+
+You are encouraged (but it is not required) to record your change into
```

Comment:
Nit; I would argue that "encouraged" implies the part in the parenthesis.

Reply:
Removed that extra bit in 46c917a90f8b3e25e1a8e30e1d410db61ab323e3